examples/networkpolicy/kubernetes-azure-hybrid.json

```diff
@@ -0,0 +1,52 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPolicy": "azure"
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "linuxpool1",
+        "count": 2,
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet",
+        "ipAddressCount": 20,
```

Comment:
see here: https://kubernetes.io/docs/admin/kubelet/

`      --max-pods int32                                          Number of Pods that can run on this Kubelet. (default 110)`

Hopefully this is an easy change to make. Just plumbing the agent pool CA address count into a new flag on kubelet. It can just default to 110 if not in CNI if that makes it easier to add this flag.